lblod/pipeline.py

```diff
@@ -0,0 +1,90 @@
+import json
+import datetime
+import os
+import uuid
+import gzip
+from rdflib import Graph
+
+import requests
+from constants import DEFAULT_GRAPH, TASK_STATUSES
+
+from lblod.file import STORAGE_PATH, construct_insert_file_query
+from lblod.harvester import collection_has_collected_files, create_results_container
+from lblod.job import update_task_status
+from sudo_query import update_sudo
+from helpers import logger
+
+
+def get_item(rdo, task):
+    # Get all possible urls
+    json_urls = {"urls": ["https://verenigingen.oscart-dev.s.redhost.be/json-ld?page=1",
+                          "https://verenigingen.oscart-dev.s.redhost.be/json-ld?page=2",
+                          "https://verenigingen.oscart-dev.s.redhost.be/json-ld?page=3"]}
+    return json.dumps(json_urls)
+
+
+def process_item(content, rdo):
+    if not os.path.exists(STORAGE_PATH):
+        os.mkdir(STORAGE_PATH)
+
+    _uuid = str(uuid.uuid4())
+    json_file_name = f"{_uuid}.json.gz"
+    json_file_path = os.path.join(STORAGE_PATH, json_file_name)
+
+    with gzip.open(json_file_path, "wt", encoding='utf-8') as f:
+        f.write(content)
```

Comment:
the rest of the harvester stack requires N-TRIPLE format to work properly. 
There is a need for either an extra  to convert  from json-ld to N-TRIPLE I think

Reply:
We also added a second service that does the conversion: https://github.com/lblod/harvesting-verenigingen-import-service